### PR TITLE
DOC-2469: Windows Repo Folder Compatibility and Template Updates

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2469

Changes:
* Replace any `<` and `>` in template file/directory names with `{` and `}` to allow cloning of `tinymce-docs` in Windows machines.
* Update template files (lots of little changes here to make it easier for future technical writers to understand how to use the templates)
* Delete unnecessary template files `antora.yml` and `supported-versions.adoc`
* Update `productminorversion` in `antora.yml`

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [-] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [-] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: -new-material-templates/release-notes-template/ROOT/partials/misc/supported-versions.adoc [hunk 1]
```diff
@@ -5,28 +5,9 @@ Supported versions of {productname}:
...
-
-* link:{legalpages}/cloud-use-subscription-agreement/[The {cloudname} Services Subscription Agreement].
-* link:{legalpages}/tiny-self-hosted-enterprise-agreement/[The {companyname} Self-Hosted Software License Agreement - (Enterprise Users)].
-* link:{legalpages}/tiny-self-hosted-oem-saas-agreement/[The {companyname} Self-Hosted Software License Agreement - (OEM & SaaS Users)].
-
-Support for Partner Plugins is provided by Partner companies and is not the responsibility of {companynameformal}. For information on Partner Plugins, see: link:{companyurl}/partners/[https://www.tiny.cloud/partners/].
-
-// This file is instructional and is never included in the published name-space.
+|7.1 |2024-05-08 |2025-11-08
+|7.0 |2024-03-20 |2025-09-20
```
Comment (kemister85) [inline -new-material-templates/release-notes-template/ROOT/partials/misc/supported-versions.adoc:12]:
```suggestion
|7.2 |2024-06-19 |2025-12-19
|7.1 |2024-05-08 |2025-11-08
|7.0 |2024-03-20 |2025-09-20
```

File: -new-material-templates/release-notes-template/antora.yml [hunk 2]
```diff
@@ -1,83 +1,14 @@
 name: tinymce
```
Comment (kemister85) [inline -new-material-templates/release-notes-template/antora.yml:1]: @FarzadHayat : What was the reason to remove the existing `antora.yml` example content?

Comment (FarzadHayat) [reply -new-material-templates/release-notes-template/antora.yml:1]: The intention was to shorten the template file to just the attributes that may need to be updated with a new major/minor release. Would you rather keep the full template?

Comment (FarzadHayat) [reply -new-material-templates/release-notes-template/antora.yml]: File has been removed as it was decided that it is no longer useful. When creating a new major release, the entire branch along with the actual `antora.yml` file will be cloned.

**Review by kemister85 — APPROVED** [review]
LGTM

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**